workflows/virology/influenza-isolate-subtyping/influenza-isolate-subtyping-test.yml

```diff
@@ -0,0 +1,236 @@
+- doc: Test outline for influenza-isolate-subtyping.ga
+  job:
+    References per segment collection:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: ref_1_pb2
+        location: https://zenodo.org/records/14620360/files/ref_1_pb2.fasta
+        filetype: fastqsanger
+      - class: File
+        identifier: ref_2_pb1
+        location: https://zenodo.org/records/14620360/files/ref_2_pb1.fasta
+        filetype: fastqsanger
+      - class: File
+        identifier: ref_3_pa
+        location: https://zenodo.org/records/14620360/files/ref_2_pa.fasta
+        filetype: fastqsanger
+      - class: File
+        identifier: ref_4_ha
+        location: https://zenodo.org/records/14620360/files/ref_4_ha.fasta
+        filetype: fastqsanger
+      - class: File
+        identifier: ref_5_np
+        location: https://zenodo.org/records/14620360/files/ref_5_np.fasta
+        filetype: fastqsanger
+      - class: File
+        identifier: ref_6_na
+        location: https://zenodo.org/records/14620360/files/ref_6_na.fasta
+        filetype: fastqsanger
+      - class: File
+        identifier: ref_7_mp
+        location: https://zenodo.org/records/14620360/files/ref_7_mp.fasta
+        filetype: fastqsanger
+      - class: File
+        identifier: ref_8_ns
+        location: https://zenodo.org/records/14620360/files/ref_8_ns.fasta
+        filetype: fastqsanger
+    Sequenced paired-end data:
+      class: Collection
+      collection_type: list:paired
+      elements:
+      - class: Collection
+        type: paired
+        identifier: SRR24839074
+        elements:
+        - identifier: forward
+          class: File
+          location: https://zenodo.org/records/14620360/files/SRR24839074_1.fastq
+          filetype: fastqsanger
+        - identifier: reverse
+          class: File
+          location: https://zenodo.org/records/14620360/files/SRR24839074_2.fastq
+          filetype: fastqsanger
+      - class: Collection
+        type: paired
+        identifier: SRR6674560
+        elements:
+        - identifier: forward
+          class: File
+          location: https://zenodo.org/records/14620360/files/SRR6674560_1.fastq
+          filetype: fastqsanger
+        - identifier: reverse
+          class: File
+          location: https://zenodo.org/records/14620360/files/SRR6674560_2.fastq
+          filetype: fastqsanger
+  outputs:
+    subtyping_results:
+      asserts:
+        has_n_lines:
+          n: 2
+        has_n_columns:
+          n: 2
```

Comment:
Subtyping *results* should also not change across WF releases. Would be great to check whether the detected subtypes meet expectations.